_data/companies/px.yaml

```
@@ -0,0 +1,7 @@
+---
+name: PX
```

What does PX stand for?

---

PX = Private Exchange, PX platform, PX inc.

---

```suggestion
name: PX (Private Exchange)
```

What do you think about this change?

---

I would prefer 'PX Platform'